frontend/src/components/Bootstrap/index.tsx

```diff
@@ -24,7 +24,7 @@ type State = {
 export default function Bootstrap() {
   const { showBoundary } = useErrorBoundary();
   const [_, setLocation] = useLocation();
-  const [state, setState] = useState<State>({ isInProgress: true });
+  const [state, setState] = useState<State>({ isInProgress: false });
```

Comment:
This will cause the bootstrap form to appear for a moment even if the bootstrap is completed.